bndtools.m2e/src/bndtools/m2e/BndConfigurator.java

```diff
@@ -118,9 +122,20 @@ public AbstractBuildParticipant getBuildParticipant(final IMavenProjectFacade pr
                     return build;
                 }
 
+                final IProject project = projectFacade.getProject();
+                IMarker[] imarkers = project.findMarkers(IJavaModelMarker.JAVA_MODEL_PROBLEM_MARKER, false, IResource.DEPTH_INFINITE);
+
+                if (imarkers != null && Arrays.stream(imarkers)
```

Comment:
Hey Ray, should we check the existing Bndtools preference for this?  That way people who want to `opt-in` can go ahead and build even in the presence of errors?

Reply:
the project settings are only accessible when the bndtools nature is associated with it... otherwise you cannot reach them.